transport.rst

```diff
@@ -91,19 +102,12 @@ Peer discovery room
 '''''''''''''''''''
 One per network. Participants can discover peers willing to open more channels. It may be implemented in the future as one presence/peer discovery room per token network, but it'd complicate the room-ownership/creation/server problem (rooms need to belong to a server. Whose server? Who created it? Who has admin rights for it?).
 
-Monitoring Service Updater Room
+Monitoring Service Updates Room
 '''''''''''''''''''''''''''''''
-Raiden nodes that plan to go offline for an extended period of time can submit a :term:`balance proof` to the Monitoring Service room. The Monitoring Service will challenge Channel on their behalf in case there’s an attempt to cheat (i.e. close the channel using earlier BP)
+Raiden nodes can submit a :term:`MonitoringRequest` to the Monitoring Service room when they go
+offline. The Monitoring Service will submit their balance proof on their behalf.
 
-Pathfinding Service Updater Room
+Pathfinding Service Updates Room
 ''''''''''''''''''''''''''''''''
-Raiden nodes can query shortest path to a node in a Pathfinding room.
-
-Direct Communication Rooms
-''''''''''''''''''''''''''
-In Matrix, users can send direct e2e encrypted messages to each other through private/invite-only rooms.
-
-Blockchain Event Rooms
-''''''''''''''''''''''
-Each operator registered with the Raiden :ref:`ServiceRegistry` could provide a room, where relevant events from Raiden Token Networks are published. E.g. signed, so that false info could be challenged.
-
+Raiden Nodes publish BalanceUpdates to the Pathfinding Service room. The Pathfinding Service can
```

Comment:
I think the nodes here send two different messages PFSCapacityUpdate and PFSFeeUpdates or so. Can you re-check with Paul or Karl?

Reply:
I added it. correct @karlb ?

Reply:
Yes, looks good, now.